clients/pyethereum:develop/pyethapp.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Immediately abort the script on any error encountered
+set -e
+
+if [ $HIVE_TESTNET ];
+then
+  # TODO: Write better sed magic?
+  sed -i "s/  ACCOUNT_INITIAL_NONCE.*/  ACCOUNT_INITIAL_NONCE: 1048576/g" /root/.config/pyethapp/config.yaml
```

Comment:
I don't think this is needed any more. Since the chainid EIP, it doesn't make sense to screw around with initial nonces.